vendor/bat-native-ads/src/bat/ads/internal/ad_events/ad_event_util.h

```diff
@@ -6,14 +6,28 @@
 #ifndef BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_AD_EVENTS_AD_EVENT_UTIL_H_
 #define BRAVE_VENDOR_BAT_NATIVE_ADS_SRC_BAT_ADS_INTERNAL_AD_EVENTS_AD_EVENT_UTIL_H_
 
+#include "bat/ads/internal/ad_events/ad_event_info.h"
```

Comment:
Seems `ad_event_info.h` include should be moved to .cc file.

Reply:
fixed